src/components/Sidebar.tsx

```diff
@@ -17,11 +18,11 @@ const Sidebar = ({menuCollapsed, onToggleSidebar}) => {
                 position: "fixed",
                 top: 0,
                 left: 0,
-                height: "100vh",
+                height: "100%",
```

Comment:
Any specific reason for that change?

Reply:
In some mobiles the side menu ended before the page, making a gap in the bottom
![image](https://user-images.githubusercontent.com/99989478/181808176-caed2fc1-3508-458f-8ffa-6c52ece92e1f.png)